src/AssemblyGenerator.Attributes.cs

```diff
@@ -12,35 +11,34 @@ public partial class AssemblyGenerator
         public void CreateCustomAttributes(EntityHandle parent, IEnumerable<CustomAttributeData> attributes)
         {
             // TODO: implement support for custom attributes
-            return;
 
-            foreach (var attr in attributes)
-            {
-                var type = attr.AttributeType;
-                GetOrCreateType(type); // create type
+            //foreach (var attr in attributes)
```

Comment:
I suggest to avoid commented-out code. Whenever commented-out code happens, there should be a comment that explain if/when the code will be uncommented, or the _why_ it is left in place.

Reply:
This code was already dead due to early return from the method. Since, the latest commit was a big refactoring of the code, some of the parts blocked the successful code compilation. Since, we would most likely to rewrite this part to support custom attributes, I didn't bother to fix them.